docs/package.json

```diff
@@ -37,7 +37,7 @@
   "devDependencies": {
     "@commercetools-frontend/babel-preset-mc-app": "21.14.3",
     "prismjs": "1.29.0",
-    "webpack": "5.74.0"
+    "webpack": "4.46.0"
```

Comment:
Just wondering, is it because `DefinePlugin` doesn't work in webpack 5?

Reply:
No, actually Storybook v5 (the one we're currently using) doesn't use Webpack v5, so we should use also v4.

<img width="447" alt="image" src="https://user-images.githubusercontent.com/1110551/200819562-dae775a9-663d-4eab-a8e1-83e1a24513ad.png">